### ISSUE TITLE
Not working with hoverOnlyWhenSupported

### ISSUE THREAD

**Issue by steida** [soorria/tailwindcss-hocus#4]
hoverOnlyWhenSupported is a Tailwind future feature to prevent showing hover on touch devices. It works, but unfortunately, not with hocus. It's probably a Tailwind bug. Leaving it here if someone has the same issue.

**Comment by steida** [comment]
Tested with:


```tsx
<Pressable className="focus:bg-red-400">
  <Text>focus:bg-red-400</Text>
</Pressable>
<Pressable className="focus-visible:bg-red-400">
  <Text>focus-visible:bg-red-400</Text>
</Pressable>
<Pressable className="hover:bg-red-400">
  <Text>hover:bg-red-400</Text>
</Pressable>
<Pressable className="hocus:bg-red-400">
  <Text>hocus:bg-red-400</Text>
</Pressable>
<Pressable className="hocus-visible:bg-red-400">
  <Text>hocus-visible:bg-red-400</Text>
</Pressable>
```

**Comment by soorria** [comment]
Pretty sure this is an issue on my end - not Tailwind's. Tailwind uses a different selector when `hoverOnlyWhenSupported` is enabled. I'll fix up in the next couple days.

**Comment by soorria** [comment]
Should be fixed in v0.0.6
